code/__DEFINES/preferences.dm

```diff
@@ -85,6 +85,9 @@
 #define EXP_TYPE_GHOST "Ghost"
 #define EXP_TYPE_ADMIN "Admin"
 
+#define EXP_TYPE_JOB_CYBORG "Cyborg"
+#define EXP_TYPE_JOB_AI "AI"
```

Comment:
These lines could be updated with the new defines
https://github.com/tgstation/tgstation/blob/4fbed650716a0bd6014d15afb30cfc247cc69bac/code/modules/jobs/jobs.dm#L91
https://github.com/tgstation/tgstation/blob/4fbed650716a0bd6014d15afb30cfc247cc69bac/code/modules/jobs/jobs.dm#L98

Reply:
Defines are no longer necessary with above code changes and have been removed.

At some point we could probably benefit from moving all job names to #defines instead of having them hardcoded everywhere in code (which makes it a nightmare if we ever changed the name of an existing job), but that's another PR for another day.